src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java

```diff
@@ -594,6 +595,8 @@ public void checkUserStatusIsActiveByUserId() {
 
     @Test(expected = AccessDeniedException.class)
     public void checkUserStatusIsActiveByUserId_Throws403_WhenNoUserFoundWithGivenId() {
+        when(professionalUserRepository.findByUserIdentifier(any(String.class))).thenReturn(null);
+
```

Comment:
prefix test and add verify for when conditions

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/620/commits/c268d76aff9fd43600c21a25a63cedcf73515b00